evap/evaluation/models.py

```diff
@@ -381,17 +381,17 @@ def all_evaluations_finished(self):
 class Evaluation(LoggedModel):
     """Models a single evaluation, e.g. the exam evaluation of the Math 101 course of 2002."""
 
-    class State:
-        NEW = 10
-        PREPARED = 20
-        EDITOR_APPROVED = 30
-        APPROVED = 40
-        IN_EVALUATION = 50
-        EVALUATED = 60
-        REVIEWED = 70
-        PUBLISHED = 80
+    class State(models.IntegerChoices):
+        NEW = 10, _("new")
+        PREPARED = 20, _("prepared")
+        EDITOR_APPROVED = 30, _("editor_approved")
```

Comment:
(Maybe follow up work)

@richardebeling @Kakadus 

The string `editor_approved` is still not intended for display on the website (we have the `statename` template filter for that) - maybe we can get rid of it at some point and just encode it here? We have other `*Choices` where we use whitespace in the string id (for example `GradeDocument.Type`). I suppose we should come up with a getter that always returns `Evaluation.State` then (not like our current `evaluation.state` which might return `int`)

Is there any reason to not replace `_` with ` ` already now? If not, we could do it now to possibly avoid an additional migration later

Reply:
> Is there any reason to not replace _ with  already now?

None that I know of, putting in the user-facing string is fine with me 👍

Reply:
searching for `_\("[\w_ ]*_[\w_ ]*"\)` does not yield any other translations, so I'm fine with fixing this here

Reply:
Note: The same is probably true for `in_evaluation`. Not having another migration doesn't seem so important to me that this PR needs to be blocked by that. I'd say this can be part of another PR.